main.tf

```diff
@@ -0,0 +1,95 @@
+terraform {
+    required_providers {
+        azurerm = {
+            source = "hashicorp/azurerm"
+            version = ">= 2.49"
+        }
+    }
+
+    backend "azurerm" {
+        resource_group_name  = "Capita1_LouisEgbufor_ProjectExercise"
+        storage_account_name = "tfstate5353"
+        container_name       = "tfstate"
+        key                  = "terraform.tfstate"
+    }
+}
+
+
+provider "azurerm" {
+    features {}
+}
+
+
+data "azurerm_resource_group" "main" {
+    name = var.resource_group       
+}
+
+resource "azurerm_app_service_plan" "main" {
+    name = "${var.prefix}-terraformed-asp"
+    location = var.location
+    resource_group_name = data.azurerm_resource_group.main.name
+    kind = "Linux"
+    per_site_scaling = false    
+    reserved = true
+    sku {
+        tier = "Basic"
+        size = "B1"
+    }
+}
+resource "azurerm_app_service" "main" {
+    name = var.webapp
```

Comment:
You're using individual input values for these names which gives you flexibility, but you might find explicitly adding the prefix to all of the names makes it easier to reason about those names, e.g. being able to leave the names the same between environments, and just vary the prefix